core/tests/test_utils.py

```diff
@@ -397,16 +446,35 @@ def set_collection_editors(self, collection_editor_usernames):
             self.set_user_role(name, feconf.ROLE_ID_COLLECTION_EDITOR)
 
     def get_current_logged_in_user_id(self):
+        """Gets the user_id of the current logged-in user.
+
+        Returns:
+            str. Value of USER_ID env variable.
```

Comment:
str. The user_id of the currently logged-in user. In tests, we simulate this using a USER_ID env variable.

Reply:
Done!